static/src/js/components/SavedProjects/__tests__/SavedProjects.test.jsx

```diff
@@ -1,142 +1,130 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-import Table from 'react-bootstrap/Table'
+import { screen } from '@testing-library/react'
 
-import Spinner from '../../Spinner/Spinner'
-import PortalLinkContainer from '../../../containers/PortalLinkContainer/PortalLinkContainer'
 import { SavedProjects } from '../SavedProjects'
-
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(overrideProps) {
-  const props = {
-    savedProjects: [],
-    savedProjectsIsLoading: false,
-    savedProjectsIsLoaded: false,
+import setupTest from '../../../../../../jestConfigs/setupTest'
+
+jest.mock('../../../util/addToast', () => ({
+  addToast: jest.fn()
+}))
+
+jest.mock(
```

Comment:
I think we can also just update this mock in `static/src/js/components/DownloadHistory/__tests__/DownloadHistory.test.jsx` to use `jest.mock('../../../containers/PortalLinkContainer/PortalLinkContainer', () => jest.fn(({ children }) => <div>{children}</div>))

instead of the needing the `__esModule: true` key sorry didn't notice it on the last Pr
`